src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/ast-search-button.html

```diff
@@ -147,3 +133,47 @@
       .catch(error => console.error('Fetch operation failed:', error));
   });
 </script>
+
+<script>
```

Comment:
Maybe, we should have a `static/js` directory where we can place all these logic. What do you think?

Reply:
I tried it, but since we're fetching the JSON file, the JavaScript wasn't working as expected. I'll give it another try.

Reply:
added the script, had to give relative path to search.json file . But all other are working as expected